### PR TITLE
Use five-color disposition in the place of the Enneagram

### PR DESCRIPTION
**Pull request by dysbulic**
At the request of @alalonde, I'm breaking #401 into smaller chunks. This chunk adds the color information, replaces the Enneagram in the profile editing section, and removes the Enneagram table.

It also includes the changes in #421 so that I could deploy a test server.

### CODE REVIEW

File: packages/web/components/Player/Section/PlayerHero.tsx [hunk 1]
```diff
@@ -1,95 +1,124 @@
...
+              mb={3}
+              textAlign="left"
+            >
+              Color Disposition
+            </Text>
+            <Link
+              isExternal
+              href={`//dysbulic.github.io/5-color-radar/#/combos/${type.mask.toString(2)}`}
+              maxH='6rem'
+            >
```
Comment (alalonde) [inline packages/web/components/Player/Section/PlayerHero.tsx:98]: External to this PR; but a bit more context would be useful in the linked page. I would expect to at least see the colors + one-word description (eg. "savagery") there.

Comment (alalonde) [reply packages/web/components/Player/Section/PlayerHero.tsx:98]: Also it seems we should link to some internal page (in the wiki / notion) that describes what the colors / disposition means. Maybe with a little question mark icon or something.

File: packages/web/components/Setup/SetupPersonalityType.tsx [hunk 2]
```diff
@@ -60,63 +63,130 @@ export const SetupPersonalityType: React.FC<SetupPersonalityTypeProps> = ({
...
+
+  // mask should always only have at most a single bit set
+  const toggleMaskElement = (mask = 0): void => {
+    setColorMask((current = 0) => {
+      if ((mask & current) > 0) {
+        // if the bit in mask is set
+        return current & ~mask; // unset it
+      }
+      return current | mask; // otherwise set it
+    });
```
Comment (alalonde) [inline packages/web/components/Setup/SetupPersonalityType.tsx:76]: nice, thanks for the comments for bitwise-frail devs like myself :D

File: packages/web/components/Setup/SetupPersonalityType.tsx [hunk 3]
```diff
@@ -60,63 +63,130 @@ export const SetupPersonalityType: React.FC<SetupPersonalityTypeProps> = ({
...
+          Person&#xAD;ality Type
+        </MetaHeading>
+        <Text mb={10}>
+          Please select your personality components below. Not sure what type
+          you are?
+          <Text as="span"> Take </Text>
+          <MetaLink
+            href="//dysbulic.github.io/5-color-radar/#/explore/"
+            isExternal
           >
```
Comment (alalonde) [inline packages/web/components/Setup/SetupPersonalityType.tsx:92]: This quiz is sweet; nice work

File: packages/web/components/Setup/SetupSkills.tsx [hunk 4]
```diff
@@ -44,6 +44,7 @@ export const SetupSkills: React.FC<SetupSkillsProps> = ({
     });
 
     if (error) {
+      console.warn(error); // eslint-disable-line no-console
```
Comment (alalonde) [inline packages/web/components/Setup/SetupSkills.tsx:47]: I dont think these are necessary. Really we should be using some logging tool like Sentry to post unexpected errors to a Discord channel... but that's for another ticket :)

File: packages/web/graphql/getPersonalityInfo.ts [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+export const images: {
+  [x: number]: string
+} = {
+  0b10000: JusticeAltImg,
+  0b01000: WisdomAltImg,
+  0b00100: AmbitionAltImg,
+  0b00010: ChaosAltImg,
+  0b00001: BalanceAltImg,
+};
```
Comment (alalonde) [inline packages/web/graphql/getPersonalityInfo.ts:31]: clever!

File: packages/web/package.json [hunk 6]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "concurrently \"next\" \"yarn generate --watch\"",
+    "serve": "concurrently \"next start\" \"watch --interval 300 'yarn build'\"",
```
Comment (alalonde) [inline packages/web/package.json:7]: why this change?

File: packages/web/utils/mathHelper.ts [hunk 7]
```diff
@@ -0,0 +1,5 @@
+// true if the number is a power of 2
+export const isPow2 = (int: number): boolean => (
+  // eslint-disable-next-line no-bitwise
```
Comment (alalonde) [inline packages/web/utils/mathHelper.ts:3]: let's just turn `no-bitwise` off in the lint config

File: yarn.lock [hunk 8]
```diff
@@ -15433,7 +15426,22 @@ ethereumjs-vm@^2.0.2, ethereumjs-vm@^2.3.4, ethereumjs-vm@^2.6.0:
     rustbn.js "~0.2.0"
     safe-buffer "^5.1.1"
 
-ethers@5.0.32, ethers@^5.0.18:
+ethers@4.0.48:
```
Comment (alalonde) [inline yarn.lock:15429]: two different versions of the same package? Not sure how this even works...

File: yarn.lock [hunk 9]
```diff
@@ -27941,6 +27949,11 @@ schema-utils@^3.0.0:
     ajv "^6.12.5"
     ajv-keywords "^3.5.2"
 
+scrypt-js@2.0.4:
```
Comment (alalonde) [inline yarn.lock:27952]: Not sure what requires this particular version of this package, but ideally it would match the version already-in-use

**Review by alalonde — APPROVED** [review]
Bunch of questions but no blockers. Maybe address the questions and then merge it!

File: packages/web/components/Player/Section/PlayerHero.tsx (hunk 1 above)
Comment (dysbulic) [reply packages/web/components/Player/Section/PlayerHero.tsx:98]: I added the MetaGame names to the [Magic quiz site](https://dysbulic.github.io/5-color-radar/#/combos/1011). That site has a link to [the definitive Magic article](https://humanparts.medium.com/the-mtg-color-wheel-c9700a7cf36d) and the [Magic wiki](https://mtg.fandom.com) for each color combination as well as some of the icons.

The color bar in the player's profile opens a link to the quiz site's [result](https://dysbulic.github.io/5-color-radar/#/combos/11011) for that combination. My idea was that they would be seeing the colors through the tests on the editing page and via a link on the display page.

The only visual indication that the bar is a link is the name gets underlined when hovered.

The four color combinations don't have a description on the wiki or in the article. I was planning on pushing the issue to the Writers once the layouts are finalized and see if there's anyone there who knows Magic well enough to write summaries for those.

File: packages/web/components/Setup/SetupPersonalityType.tsx (hunk 3 above)
Comment (dysbulic) [reply packages/web/components/Setup/SetupPersonalityType.tsx:92]: ¿The first one with the sliders made sense? I thought it shouldn't take much explanation, but you never know with a UI how intuitive others will find it.

File: packages/web/components/Setup/SetupSkills.tsx (hunk 4 above)
Comment (dysbulic) [reply packages/web/components/Setup/SetupSkills.tsx:47]: I'd be fascinated to use some sort of centralized logging system. Since Vercel and Github are getting Hasura timeouts, I assume whatever is generating the production site is getting them as well.

File: packages/web/package.json (hunk 6 above)
Comment (dysbulic) [reply packages/web/package.json:7]: The `dev` command was running out of memory on the smallest render.com instance, so I was trying to mimic the behavior of production which is static but intermittently regenerated.

I can pull it out, it isn't being used by anyone including myself.

File: packages/web/utils/mathHelper.ts (hunk 7 above)
Comment (dysbulic) [reply packages/web/utils/mathHelper.ts:3]: Done.

File: yarn.lock (hunk 8 above)
Comment (dysbulic) [reply yarn.lock:15429]: `yarn list` shows:

```
├─ @metafam/backend@0.1.0
│  ├─ ethers@5.0.32
├─ @metafam/discord-bot@0.1.0
│  ├─ ethers@4.0.48
├─ @metafam/utils@1.0.0
│  ├─ ethers@5.0.32
├─ @metafam/web@0.1.0
│  ├─ ethers@5.0.32
```
¿So, I'd update the discord bot?

Comment (dysbulic) [reply yarn.lock:15429]: Yes. That is what I'd do.

File: packages/web/package.json (hunk 6 above)
Comment (dysbulic) [reply packages/web/package.json]: Done.

File: yarn.lock (hunk 9 above)
Comment (dysbulic) [reply yarn.lock]: Fixed.

File: yarn.lock (hunk 8 above)
Comment (alalonde) [reply yarn.lock:15429]: Good catch! I actually noticed the same thing on my discord branch after I posted this comment..

**Comment by dysbulic** [comment]
@alalonde, I got Hasura's metadata woes settled and got it to start eventually.

This should now be ready to deploy. 🤞🏼

**Comment by alalonde** [comment]
Could you please rebase this from master again? Still some conflicts it seems.
